src/styl/home.styl

```diff
@@ -1,9 +1,39 @@
 .home
-  padding 10px
+  padding-top 5px
+  >*
```

Comment:
I'd try to avoid that, by principle. '*' in css rule is highly inefficient.

Reply:
Done.